01week/datatypes.js

```diff
@@ -0,0 +1,58 @@
+"use strict"
+
+//1. Write a JavaScript program to display the current day and time
+function getDateAndTime(){
+  return new Date()
+}
+getDateAndTime()
+
+//2. Write a JavaScript program to convert a number to a string
+function changeNumToStr(num){
+  return num.toString()
+}
+changeNumToStr(9)
+
+//3. Write a JavaScript program to convert a string to the number.
+function changeStrToNum(str){
+  return Number(str)
+}
+changeStrToNum("9")
+
+//4. Write a JavaScript program that takes in different datatypes and prints out whether they are a:
+// Boolean
+// Null
+// Undefined
+// Number
+// NaN
+// String
+function whatDataTypeIsIt(data){
+  if(data===null){return 'Null'}
+  else if(Number.isNaN(data)){return 'NaN'}
+  else{return typeof data}
+}
+whatDataTypeIsIt(NaN)
+
+//5. Write a JavaScript program that adds 2 numbers together.
+function sumTwoNums(num1,num2){
+  return num1 + num2
+}
+sumTwoNums(4,5)
+
+//6. Write a JavaScript program that runs only when 2 things are true.
+function twoTruths(thing1,thing2){
+  if(thing1 == true & thing2 == true){return 'Both are true.'}
```

Comment:
This evaluates if both things are equal to the boolean true. I need it to evaluate if both things are truthy in nature.